packages/itwinui-react/src/core/Toast/Toast.tsx

```diff
@@ -169,27 +175,35 @@ export const Toast = (props: ToastProps) => {
     return { translateX, translateY };
   };
 
+  const motionOk = useMediaQuery('(prefers-reduced-motion: no-preference)');
+
   return (
     <Transition
       timeout={{ enter: 240, exit: animateOutTo ? 400 : 120 }}
       in={visible}
       appear={true}
       unmountOnExit={true}
       onEnter={(node: HTMLElement) => {
-        node.style.transform = 'translateY(15%)';
-        node.style.transitionTimingFunction = 'ease';
+        if (motionOk) {
+          node.style.transform = 'translateY(15%)';
```

Comment:
Because this `translateY` doesn't happen anymore on when reduced motion is active, the screenshots will be slightly altered as they were taking during this translation in `onEnter`, but not after the transformation in `onEntered`.

Reply:
you could try an alternate approach where instead of conditionally applying styles, you could conditionally set the transition duration, i.e. something like "normal duration if motion ok. 0.001s if reduced motion preferred".

doing that might avoid the issues with positioning.

Reply:
Do the tests run with reduced motion on? Huh

Reply:
I'm mistaken. These tests do not run with reduced motion on. The toasts are not transitioning properly when reduced motion is off (the default). I'll work on a fix

Reply:
These image changes have been reverted aside from one. There is a slight horizontal change in the toast buttons on the Position Changed story. I'm not sure why